app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -385,6 +395,88 @@ class HomeActivityTest {
     }
   }
 
+  // TODO(#2057): Remove when TextViews are properly measured in Robolectric.
+  @RunOn(TestPlatform.ESPRESSO) // Incorrectly passes on Robolectric and shouldn't be re-enabled
+  @Config(qualifiers = "port-mdpi")
+  @Test
+  fun testHomeActivity_phonePort_longProfileName_displaysWelcomeMessageCorrectly() {
```

Comment:
testHomeActivity_longProfileName_welcomeMessageIsDisplayed

Reply:
`_welcomeMessageIsDisplayed` for tablet ones too

Reply:
Done